doc/project/checklist.md

```diff
@@ -453,6 +453,7 @@ Any waiver files have been reviewed.
 ### PRE_VERIFIED_SUB_MODULES_V2
 
 Sub-modules that are pre-verified with their own testbenches have already reached V2 or a higher stage.
+List these sub-modules in the dv_doc and the coverage of these sub-modeules can be excluded in the IP-level testbench.
```

Comment:
Nit, could we also have some place where we list the specific pre-verified prim modules? Then it would be easy to create this list for each IP.

Reply:
we can find these in the [dashboard](https://reports.opentitan.org/hw/top_earlgrey/dv/summary.html) of regression results. They are all named as prim_*.